cypress/e2e/teampage.cy.js

```diff
@@ -0,0 +1,95 @@
+describe('Chosing Drivers Path', () => {
+  beforeEach('', () => {
+    cy.intercept('https://v1.formula-1.api-sports.io/rankings/drivers?season=2021', {
+      fixture: 'allDrivers.json'
+    })
+      .intercept('https://v1.formula-1.api-sports.io/drivers?name=Lewis%20Hamilton', {
+        fixture: 'lewis.json'
+      })
+      .intercept('https://v1.formula-1.api-sports.io/drivers?name=Max%20Verstappen', {
+        fixture: 'max.json'
+      })
+      .intercept('https://v1.formula-1.api-sports.io/rankings/teams?season=2021', {
+        fixture: 'teams2021.json'
+      })
+      .visit('http://localhost:3000/team')
+  });
+
+  it('should ask the user to choose one driver at a time', () => {
+    cy.get('.driver-select').contains('Pick your driver:')
+      .get('select').select('Lewis Hamilton')
+      .get('.submit-driver').click()
+      .get('.driver-select').should('exist');
+  });
```

Comment:
To test that it's asking to choose one at a time, consider adding a line to check that the user cannot select another driver before submitting the first or that there's an instructional line somewhere. As it stands, you are not testing that the site "is asking the user" only that the user is able to select a driver. Line 22 also seems strange because we already know that '.driver-select' exists. Consider checking instead that the value of the select has been cleared out and is ready for a second selection.

Reply:
Ooh great point!  I really struggled with this test because the second div doesn't appear until the first is filled, but since it's the same component the class name I'm getting is the same both times & it got pretty confusing.  Checking the select value is a good addition for sure

Reply:
Ahhh I was wondering if there are two divs! A easy workaround then would most likely to be to check the quantity of the divs and ensure that it's one at first and then two after submitting!